nsv13/code/modules/overmap/fighters/heavy_build.dm

```diff
@@ -0,0 +1,562 @@
+#define HBS_CHASSIS							0

```

Comment:
Make this one sec of generic fighter build defines. Building fighters shouldn't vary much apart from the chassis (See: mecha construction)

Reply:
Each one has a different build path.